extension/llm/export/partitioner_lib.py

```diff
@@ -140,7 +140,7 @@ def get_qnn_partitioner(
 
     return QnnPartitioner(  # pyre-fixme[16]
         generate_qnn_executorch_compiler_spec(  # pyre-fixme[16]
-            soc_model=QcomChipset.SM8650,  # default to SM8650  # pyre-fixme[16]
+            soc_model=QcomChipset.SM8450,  # default to SM8450  # pyre-fixme[16]
```

Comment:
To unblock benchmarking, let's updating the default for S22. Will follow-up in a separate PR to make it configurable. cc: @shewu-quic

Reply:
ugh do you explicitly need this one?

Reply:
Chipset mismatch may lead to weird runtime issue per https://github.com/pytorch/executorch/issues/4973. I want to avoid debugging this kind of issue so yes have to make sure the chipset matches the target device.

> benchmark qnn + llama

Not only llama but also other non-genai models as well

Reply:
I'd prefer to leave this line as is, until we make it configurable.

Reply:
FYI. Follow-up PR in https://github.com/pytorch/executorch/pull/5100. Let's move further discussion regarding chipset there